com.sap.adt.abapcleaner/src/com/sap/adt/abapcleaner/programbase/CommandLineArgs.java

```diff
@@ -6,6 +6,8 @@
 public class CommandLineArgs {
 	private static final String LINE_SEP = System.lineSeparator();
 
+	private static final String OPT_SOURCE_FOLDER = "--sourcefolder";
```

Comment:
no, sorry: Could we call these options and variables "source dir" and "target dir"? Because in ABAP cleaner, "folder" could actually just mean the "subfolder\" inside of some directory, whereas "dir" (or "directory") is used for the complete directory path C:\any\dir\